src/pages/Components/Notification/code.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+```css
+@import 'settings-tools/_all-settings';
+@import 'components/_c.notification';
+```
+
+## Basic usage
+
+Wrap a h3, p and a a or button tags in a div and apply the `mc-notification` class:
+
+```html
+  <div class="mc-notification">
+    <h3>Notification title</h3>
+    <p>
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.
+    </p>
+    <a href="#">Link</a>
+  </div>
+```
+
+<preview path="src/pages/Components/Notification/previews/notification"></preview>
```

Comment:
The icon size is 2mu (32px).
The title seems strong. Have you put the title in *bold* ? It must be in *semi-bold*.

Reply:
The icon size was wrong, indeed, it'll be fixed in the next commit.
About the title I used the "semi-bold" config which sets the font-weight to 600, and that's corresponds to the font-weight found on design schema, so I suppose It's right.